### ISSUE TITLE
SFI Effort: ASP.NET / IIS / EF All Repos

### ISSUE THREAD

**Issue by wadepickett** [dotnet/AspNetCore.Docs#33229]
**Parent tracking for all SFI efforts for all ASP.NET / IIS / EF repos.**

This effort will likely span more than one monthly sprint.  It is expected however that we start triaging right away.

NOTE ON SCHEDULE for just **GUID** hits:  We do not have specifics for addressing GUID hits yet as of 9/3/24.  I am noting it here for the tracking item for GUIDS so we don't forget it and look again to see if we have more details, but it may or may not be something we handle in September. The plan is to use DocuTune to remediate as many of the GUID hits as possible and it may be that there is no work to do on GUID's as a result.

For details see:
[SFI info in Teams](https://teams.microsoft.com/l/message/19:49ff5836203f4b358334bfcfa593f252@thread.tacv2/1722361260037?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=8d8475c9-f864-4ed0-825e-b2e419026017&parentMessageId=1722361260037&teamName=Content%20Team%20%7C%20C%2BE%20Skilling&channelName=General&createdTime=1722361260037)
•  [ROPC & Global - SFI Update - Content remediation action items!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fteams.microsoft.com%2Fl%2Fmessage%2F19%3A49ff5836203f4b358334bfcfa593f252%40thread.tacv2%2F1722361260037%3FtenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47%26groupId%3D8d8475c9-f864-4ed0-825e-b2e419026017%26parentMessageId%3D1722361260037%26teamName%3DContent%2520Team%2520%257C%2520C%252BE%2520Skilling%26channelName%3DGeneral%26createdTime%3D1722361260037&data=05%7C02%7Cwpickett%40microsoft.com%7Cf49c51fc14f747fbb02708dcb8918170%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638588182292262817%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=qFwA7q8Qb3K8nMotMj4%2FasBUwmbPwDmQ7pMyFLmMZIA%3D&reserved=0) (ROPC/Global Admin) July 30, 2024 10:41 AM
•  [Images - SFI update: action items inside!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fteams.microsoft.com%2Fl%2Fmessage%2F19%3A49ff5836203f4b358334bfcfa593f252%40thread.tacv2%2F1723026587663%3FtenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47%26groupId%3D8d8475c9-f864-4ed0-825e-b2e419026017%26parentMessageId%3D1723026587663%26teamName%3DContent%2520Team%2520%257C%2520C%252BE%2520Skilling%26channelName%3DGeneral%26createdTime%3D1723026587663&data=05%7C02%7Cwpickett%40microsoft.com%7Cf49c51fc14f747fbb02708dcb8918170%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638588182292267776%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=7%2F8Lgw2gyh0k%2BdsAshDEU%2FTiz69vvZPs25%2BaLNQt8t8%3D&reserved=0) (Images) August 7, 2024 3:29 AM

CS effort for repo:
**AspNetCore.Docs**

- [x] GUIDS:

- [x] 0: Done, all good here.

- [ ] ROPC: 66 left to fix.   3 fixed so far with PR #33408 which provided a standard example of how we might fix many of these.  Triage remaining and fix what can be quickly fixed with include statement, see tracking item #33504

- [x] Global: 0: Done, all good here.

- [ ] Image scans: 59 images
    - [x] Wade has checked for orphaned images, found 15 of the 59.  Removed with PR #33501
    - [x] Wade found 3 local dev trust cert dialog images that will be will not fix (Completed - no change)
    - [ ] The remaining 41 images Wade will edit to remove strings or changes strings in the image.

CS effort for repo:
**AspNetDocs**

- [ ] GUIDS: 2 files  (9/3/24 note: Do not fix GUIDS, this may be a auto fix with docutune.)

- [ ] Global: 3 Files

- [ ] ROPC: 86 files

- [ ] Image scans: 859 images
  

CS effort for repo:
**IIS-docs and MicrosoftDocs/IIS.Administration-docs**

GUIDS: 0: Done All good here for both repos.

ROPC:

- [ ] 107 files in IIS-pr

- [x] 0 in IIS.Administration-docs

- [x] Global: 0: Done All good here for both repos.

Image scans:

- [ ] 512 images in IIS-docs

- [ ] 1 image in IIS.Administration-docs

CS effort for repos:
**dotnet_EntityFramework_Docs
dotnet_EntityFramework_ApiDocs**

- [x] GUIDS: 0: Done. All good here for both repos.

- [x] Global: 0: Done. All good here for both repos.

ROPC:

- [ ] 35 Files in EntityFramework.Docs

- [ ] 0 in EntityFrameworkApiDocs

Image scans:

- [ ] 10 images in EntityFramework.Docs

- [x] 0 images in EntityFramework_ApiDocs

---
[Associated WorkItem - 302690](https://dev.azure.com/msft-skilling/Content/_workitems/edit/302690)

**Comment by Rick-Anderson** [comment]
@wadepickett  add internal links to the CS effort, what's required, etc.

**Comment by wadepickett** [comment]
I was keeping those internal, you have them.  But if you want them here I will.

**Comment by wadepickett** [comment]
@tdykstra and @Rick-Anderson,   I thought the new labels sfi-ropc and sfi-admin would show up by now and was intending to assign them for the individual "sfi-" related GitHub issues for global, ropc, etc.   Andy PR'd a few days ago.  But they don't seem to be available yet.  Are we waiting until later to make them available or maybe I am missing something embarrassingly obvious?

Andy added these a few days ago:
        {
            "Label": "sfi-ropc",
            "ParentNodeId": 271716
        },
        {
            "Label": "sfi-admin",
            "ParentNodeId": 271716
        }

**Comment by wadepickett** [comment]
Please disregard, I see the labels just needed to be added through the GitHub UI.  done.

**Comment by wadepickett** [comment]
Reminder: For details see:
[SFI info in Teams](https://teams.microsoft.com/l/message/19:49ff5836203f4b358334bfcfa593f252@thread.tacv2/1722361260037?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=8d8475c9-f864-4ed0-825e-b2e419026017&parentMessageId=1722361260037&teamName=Content%20Team%20%7C%20C%2BE%20Skilling&channelName=General&createdTime=1722361260037)

**Comment by Rick-Anderson** [comment]
@wadepickett  @tdykstra  container issues like this shouldn't have reQUEST  so I removed seQUESTered and added `mapQuest`. I don't think `mapQuest` is currently working.

**Comment by Rick-Anderson** [comment]
@wadepickett  closing as we've completed the triage.

**Comment by wadepickett** [comment]
@Rick-Anderson 
"Closing".  That is fine.  Originally it was just created as a parent to keep track of everything involved with this effort, not just triage.  So I stated up front at the very top "This effort will likely span more than one monthly sprint."  However, I should not have set a "ReQuest" label, you are right, it should not have been represented in ADO.

**Comment by tdykstra** [comment]
> @wadepickett @tdykstra container issues like this shouldn't have reQUEST so I removed seQUESTered and added `mapQuest`. 

So should we delete the [associated AzDO work item 302690](https://dev.azure.com/msft-skilling/Content/_workitems/edit/302690)?

**Comment by wadepickett** [comment]
@Rick-Anderson & @tdykstra 

We decided at some point I was handling all the Images correct?  It seems to me we did, which I am fine doing and working through right now starting at the the most severe (sev 0) in every repo and working down from there.  I can figure those out, but for ROPC I wasn't always sure what our correct answer should be.

...Going through all the rest of the SFI issues I created to make sure they are cleaned up and represent how we ended up doing things.
